src/app/cli/src/coda.ml

```diff
@@ -237,11 +237,27 @@ let daemon logger =
          ~transport:(Logger.Transport.stdout ()) ;
        (* 512MB logrotate max size = 1GB max filesystem usage *)
        let logrotate_max_size = 1024 * 1024 * 512 in
-       Logger.Consumer_registry.register ~id:"raw_persistent"
+       Logger.Consumer_registry.register ~id:"default"
+         ~processor:(Logger.Processor.raw ())
+         ~transport:
+           (Logger.Transport.File_system.dumb_logrotate ~directory:conf_dir
+              ~log_name:"coda.log" ~max_size:logrotate_max_size) ;
+       Logger.Consumer_registry.register ~id:"prover"
+         ~processor:stdout_log_processor
+         ~transport:(Logger.Transport.stdout ()) ;
+       Logger.Consumer_registry.register ~id:"prover"
+         ~processor:(Logger.Processor.raw ())
+         ~transport:
+           (Logger.Transport.File_system.dumb_logrotate ~directory:conf_dir
+              ~log_name:"coda-prover.log" ~max_size:logrotate_max_size) ;
+       Logger.Consumer_registry.register ~id:"verifier"
+         ~processor:stdout_log_processor
+         ~transport:(Logger.Transport.stdout ()) ;
+       Logger.Consumer_registry.register ~id:"verifier"
```

Comment:
I wonder if the registration for the prover and verifier should happen in `Prover.create` and `Verifiier.create` ?